pyhealth/datasets/base_ehr_dataset.py

```diff
@@ -122,7 +124,7 @@ def __init__(
             logger.debug(
                 f"Loaded {self.dataset_name} base dataset from {self.filepath}"
             )
-            self.patients = load_pickle(self.filepath)
+            self.patients, self.code_vocs = load_pickle(self.filepath)
```

Comment:
we should use "try-except" for our earlier users in case they have local pickle cache.

Reply:
Maybe do this:

```python
try:
    self.patients, self.code_vocs = load_pickle(self.filepath)
except:
    raise ValueError("Please refresh your cache by set refresh_cache=True")
```